Formula/vim.rb

```diff
@@ -16,6 +16,7 @@ class Vim < Formula
   option "override-system-vi", "Override system vi"
   option "disable-nls", "Build vim without National Language Support (translated messages, keymaps)"
   option "with-client-server", "Enable client/server mode"
+  option "enable-termtruecolor", "Include support for 24-bit colors for compatible terminals"
```

Comment:
And perhaps more importantly: If this gracefully degrades for terminals without 24-bit color support, is there any harm in making it the default instead of introducing yet another build option?